server/common/auth/index.js

```diff
@@ -3,12 +3,26 @@
 const { auth: config = {} } = require('../../config');
 const { ExtractJwt, Strategy } = require('passport-jwt');
 const { User } = require('../database');
+const LocalStrategy = require('passport-local');
 const passport = require('passport');
 
+const options = {
+  usernameField: 'email',
+  session: false
+};
+
+passport.use(new LocalStrategy(options, (email, password, done) => {
+  return User.findOne({ where: { email } })
+    .then(user => user && user.authenticate(password))
+    .then(user => done(null, user || false))
+    .error(err => done(err, false));
+}));
+
 const jwtOptions = {
   ...config,
   jwtFromRequest: ExtractJwt.fromAuthHeaderWithScheme(config.scheme),
-  secretOrKey: config.secret
+  secretOrKey: config.secret,
+  audience: 'access'
```

Comment:
Kudos for making JWT more specific, but I have two nerdy/nitpicky concerns:
1. Looking at the [JWT RFC](https://tools.ietf.org/html/rfc7519#section-4.1.3) I would argue that audience (`aud`) is not the best fit for the your use-case. Audience should be "the recipients that the JWT is intended for" which in our case is the same as the issuer (`iss`). We could stretch this definition by saying this is part of our API and the audince for this token but for me it seems far fetched. I think more proper way would be with out custom claim.
2. That being said, if we are to use optional JWT claims, which I'm all for, shouldn't we validate tokens based on them?